packages/server/knexfile.js

```diff
@@ -19,7 +19,7 @@ function walk(dir) {
   return results
 }
 
-let migrationDirs = walk('./modules')
+let migrationDirs = walk(path.resolve(__dirname, './modules'))
```

Comment:
Had to fix this otherwise knex couldn't find these directories if you invoked it from somewhere other than the `server` package root